geo-types/src/coordinate.rs

```diff
@@ -30,7 +30,18 @@ pub struct Coordinate<T: CoordNum> {
     pub y: T,
 }
 
+impl<T: CoordNum> Coordinate<T> {
+    /// Create a new instance of a coordinate.
+    /// **ATTENTION** Use [`coord!`] macro or one of the [`from`] methods instead.
+    #[inline]
+    #[doc(hidden)]
+    pub fn new__(x: T, y: T) -> Self {
+        Self { x, y }
+    }
+}
+
 impl<T: CoordNum> From<(T, T)> for Coordinate<T> {
+    #[inline]
```

Comment:
These are good inlines IMO!  Thanks!

Reply:
Does `#[inline]` even do much for generic code?

Reply:
@lnicola that's a good question - but would require too substantial of a research to pinpoint why adding these inlines boosts performance. I think it is far easier to just add inline everywhere it seems relevant (tiny code, frequent use).